packages/autorest.python/test/azure/legacy/Expected/AcceptanceTests/CustomUrlPaging/generated_samples/paging_get_pages_partial_url.py

```diff
@@ -0,0 +1,30 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from custombaseurlpaging import AutoRestParameterizedHostTestPagingClient
+
+"""
+# PREREQUISITES
+    pip install autorestparameterizedhosttestpagingclient
+# USAGE
+    python paging_get_pages_partial_url.py
+"""
+
+
+def main():
+    client = AutoRestParameterizedHostTestPagingClient()
+
+    response = client.paging.get_pages_partial_url(
+        account_name="testaccount",
+    )
+    response = [item for item in response]
```

Comment:
is this line necessary? maybe below is more clear if we want to demonstrate the response is iterable?
~~~
for item in response:
    print(item)
~~~

Reply:
it is equal to `response = [item for item in response]` or just `response = list(response)`. I use this kind of format for here: https://github.com/Azure/autorest.python/blob/d94843f4b088fb4394dd9ce984f8561e3ea721b5/packages/autorest.python/autorest/codegen/serializers/sample_serializer.py#L120-L128

Reply:
i agree with @changlong-liu here, we default remove the benefits of even returning a paging operation in the sample if we immediately read it all into memory. I know we do this for tests, but I think for samples we should iterate in a loop and not read it all into memory

Reply:
@iscai-msft Got it and have updated it.